hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -50,6 +50,7 @@
     public static String TRIP_EXAMPLE_SCHEMA = "{\"type\": \"record\","
             + "\"name\": \"triprec\","
             + "\"fields\": [ "
+            + "{\"name\": \"timestamp\",\"type\": \"double\"},"
```

Comment:
there might be some tests which are sensitive to size of records.. Please make sure all the tests pass..